.changeset/five-lobsters-juggle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/combobox': patch
+'@twilio-paste/core': patch
+---
+
+[Combobox]: Small fix for element naming pattern.
```

Comment:
It matters, the changelog this adds to is public record. It should read to inform the consumer of the library what happened

Reply:
Yeah I have a todo list item to look up some examples of really good change log notes because my drafts definitely leave room for improvement. Will push an update to this.